.circleci/config.yml

```diff
@@ -73,8 +73,8 @@ jobs:
             - run:
                 command: |
                     export IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable")
-                    OPERATOR_TAG="${IMAGE_TAG}-ubi8-${CIRCLE_SHA1}"
-                    MONITOR_TAG="${IMAGE_TAG}-ubi8-${CIRCLE_SHA1}"
+                    OPERATOR_TAG="${IMAGE_TAG}-ubi8-${CIRCLE_SHA1:0:8}"
```

Comment:
TIL

Reply:
Yes, it takes the first 8 characters of the variable